client/src/amplify-ui/styles/media.ts

```diff
@@ -21,3 +21,6 @@ export const MQDesktop = createDeviceMQ(Breakpoint.DESKTOP);
 export const MQMonitor = createDeviceMQ(Breakpoint.MONITOR);
 
 export const MAX_WIDTH = "90rem";
+
+export const isWiderThanTablet = () =>
```

Comment:
For your consideration: This file is a little confusing to me. What's the difference between `MONITOR` and `DESKTOP` for example, and `isWiderThanTablet` is kind of confusing to me given tablets operate in portrait and landscape, with vastly different aspect ratios.

Reply:
You're correct! In that case, we should rename the properties of the `Breakpoint` enum.